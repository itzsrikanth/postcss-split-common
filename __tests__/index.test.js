const path = require('path');
const postcss = require('postcss');

const plugin = require('..');

async function run() {
  const stylePaths = path.join(__dirname, 'fixtures');
  const result = await postcss([
    plugin({
      stylePaths,
      outputDir: stylePaths
     })
  ]).process('', {
    from: undefined
  });
  expect(result.css).toMatchSnapshot();
  expect(result.warnings()).toHaveLength(0);
}

describe('split common rules', () => {
  it('do nothing', async () => {
    await run();
  });
});
