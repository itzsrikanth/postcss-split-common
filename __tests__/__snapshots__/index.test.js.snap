// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`split common rules do nothing 1`] = `
"

@media screen {
  .flex {
    display: flex;
  }
}

@media screen {
}

.footer {
  background: --var(color-primary);
}"
`;
